ci/vm/provision2.sh

```diff
@@ -59,11 +39,35 @@ xpmem_load() {
   fi
 }
 
+install_gcc() {
+  # Find the GCC ver used to compile the running kernel
+  GCC_VER=$(awk -F ' ' '{print $7}' /proc/version | cut -d'-' -f1)
+  echo "INFO: Installing GCC ver $GCC_VER"
+  wget -q https://ftp.gnu.org/gnu/gcc/gcc-"$GCC_VER"/gcc-"$GCC_VER".tar.xz
+  wget -q https://ftp.gnu.org/gnu/gcc/gcc-"$GCC_VER"/gcc-"$GCC_VER".tar.xz.sig
+  wget -q https://ftp.gnu.org/gnu/gnu-keyring.gpg
+  gpgv --keyring ./gnu-keyring.gpg gcc-"$GCC_VER".tar.xz.sig gcc-"$GCC_VER".tar.xz
+  tar xf gcc-"$GCC_VER".tar.xz
```

Comment:
Does it work without `z` option?

Reply:
It's 'xz' format instead of 'gz' - much better compression.

Reply:
For .xz we need `tar xJf` - strange how it works without `J`